src/components/views/right_panel/RoomHeaderButtons.tsx

```diff
@@ -141,6 +163,12 @@ export default class RoomHeaderButtons extends HeaderButtons<IProps> {
                 isHighlighted={this.isPhase(RightPanelPhases.PinnedMessages)}
                 onClick={this.onPinnedMessagesClicked}
             />
+            <TimelineCardHeaderButton
```

Comment:
At the moment, this seems to show the header button in every room. It should only appear in rooms with a maximised widget.

Reply:
This is handled in: https://github.com/matrix-org/matrix-react-sdk/pull/7114
I thought it makes sense to bundle that logic in the context of the other PR

Reply:
Ah okay, seems okay to bundle it that way if you want to.